python/cuml/cuml/linear_model/ridge.pyx

```diff
@@ -377,3 +377,22 @@ class Ridge(UniversalBase,
 
     def get_attr_names(self):
         return ['intercept_', 'coef_', 'n_features_in_', 'feature_names_in_']
+
+    def _should_dispatch_cpu(self, func_name, *args, **kwargs):
+        """
+        Dispatch to CPU implementation when:
+        1. Multi-target regression is detected (y has more than 1 column or is 2D)
+        2. For fit-related functions only (fit, fit_transform, fit_predict)
```

Comment:
I read lists of conditions as ORed together. IIUC the logic here is just saying "dispatch to CPU for fit functions when y is 2D"? If so, I'd find this a bit more clear.

```suggestion
        Dispatch to CPU implementation when for `fit`-functions if `y` is 2D.
```

Reply:
I've simplified the doc-string.